client/client-multi/client-cpp/kaa/configuration/manager/ConfigurationManager.hpp

```diff
@@ -70,6 +70,8 @@ class ConfigurationManager : public IConfigurationManager,
         return configurationHash_;
     }
 
+    virtual ~ConfigurationManager() noexcept {}
```

Comment:
I got this error:
In file included from /Users/vasiliy/Works/Temp/2016.08.16/CppEventDemo/libs/kaa/kaa/KaaClient.hpp:37:0,
                 from /Users/vasiliy/Works/Temp/2016.08.16/CppEventDemo/libs/kaa/impl/Kaa.cpp:21:
/Users/vasiliy/Works/Temp/2016.08.16/CppEventDemo/libs/kaa/kaa/configuration/manager/ConfigurationManager.hpp:42:7: error: looser throw specifier for 'virtual kaa::ConfigurationManager::~ConfigurationManager()'
In file included from /Users/vasiliy/Works/Temp/2016.08.16/CppEventDemo/libs/kaa/kaa/configuration/manager/ConfigurationManager.hpp:24:0,
                 from /Users/vasiliy/Works/Temp/2016.08.16/CppEventDemo/libs/kaa/kaa/KaaClient.hpp:37,
                 from /Users/vasiliy/Works/Temp/2016.08.16/CppEventDemo/libs/kaa/impl/Kaa.cpp:21:
/Users/vasiliy/Works/Temp/2016.08.16/CppEventDemo/libs/kaa/kaa/configuration/manager/IConfigurationManager.hpp:72:13: error:   overriding 'virtual kaa::IConfigurationManager::~IConfigurationManager() noexcept (true)'